docs/platform.rst

```diff
@@ -82,6 +84,35 @@ The typical things an engine needs to handle are:
   same base class, apps can call methods on the engines to for example create UIs. It is up to each
   engine to implement these methods so that they work nicely inside the host application.
 
+- A startup interface for related DCC applications.
+
```

Comment:
I think we should include a couple of sentences with background/motivation here. I.E explain/link to docs about bootstrap vs. classic startup. Also, quick sentence about why this interface needed.